x-pack/plugin/blob-cache/src/test/java/org/elasticsearch/blobcache/BlobCacheUtilsTests.java

```diff
@@ -19,4 +21,10 @@ public void testReadSafeThrows() {
         final int remaining = randomIntBetween(1, 1025);
         expectThrows(EOFException.class, () -> BlobCacheUtils.readSafe(BytesArray.EMPTY.streamInput(), buffer, 0, remaining));
     }
+
+    public void testToPageAlignedSize() {
+        long value = randomLongBetween(0, Long.MAX_VALUE / 2);
```

Comment:
Should this be better testing numbers from Long.MAX_VALUE / 2 to MAX_VALUE, so it tests numbers that do not fit in an integer? Or even better a general random positive Long, so it tests all?

Reply:
I believe Long.MAX_VALUE/2 already does not fit an integer by 31 bits? I opted to not go near `Long.MAX_VALUE` border, since it seemed unnecessary, but let me tweak this to a higher number for sure.

Reply:
Done in 579606891f0943f4d7ea72f7acdb0b0e1bc8a709

Reply:
Wrote it badly. That's what I get from writing on mobile while in conference! Thanks for making it higher.